lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -718,7 +718,7 @@ function dosomething_helpers_get_current_user($user = NULL, $asArray = FALSE) {
   }
 
   // Authenticated User
-  $northstar_user = dosomething_northstar_get_user($user->uid);
+  $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
```

Comment:
@angaither Update coming! You're too quick! (it's why I had the "work in progress"... just had a bunch of stuff changing and wanted to push up commits bit by bit so I don't lose track 😬 )